frontend/package.json

```
@@ -46,6 +50,7 @@
     "test": "react-scripts test",
     "eject": "react-scripts eject",
     "lint": "eslint -c .eslintrc.yml --ext .js,.ts,.tsx src/ ../app/electron",
+    "format": "prettier --write src/ ../app/electron",
```

Just running `npm run format` wasn't picking up the config.

```suggestion
    "format": "prettier --config .prettierrc --write src/ ../app/electron",
```